besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1656,6 +1659,10 @@ private GenesisConfigFile readGenesisConfigFile() {
             ? GenesisConfigFile.fromSource(genesisConfigSource(genesisFile))
             : GenesisConfigFile.fromResource(
                 Optional.ofNullable(network).orElse(MAINNET).getGenesisFile());
+    if (network.equals(EPHEMERY)) {
+      ephemeryGenesisFile = new EphemeryGenesisFile();
+      ephemeryGenesisFile.updateGenesis();
+    }
```

Comment:
see comment on EphemeryGenesisFile class - think it needs to return the GenesisConfigFile with the ephemery overrides applied eg
effectiveGenesisFile = ephemeryGenesisFile.updateGenesis()

instead of modifying the EPHEMERY object